### PR TITLE
Fixes for table component

### PR DESCRIPTION
**Pull request by chormaz**
# Description

Fixes to the new table component after testing. Corrected some UI-issues.

Completes: [AB#82130](https://statoildeveloper.visualstudio.com/b6d97460-2e53-47ae-aad4-b41396d90828/_workitems/edit/82130)

# Checklist:

- [x ] I have performed a self-review of my own code.
- [x] I have linked my DevOps task using the AB# tag.
- [x ] My code is easy to read, and comments are added where needed.
- [x ] My code is covered by tests.

### CODE REVIEW

File: src/modules/Preservation/views/AddScope/SelectTags/SelectTags.tsx [hunk 1]
```diff
@@ -11,6 +11,53 @@ import ProcosysTable from '@procosys/components/Table';
...
+                <OverflowColumn>{tagRow.description}</OverflowColumn>
+            </Tooltip>
+        </div>
+    );
+};
+
+const getMccrResponsiblesColumn = (row: TableOptions<TagRow>): JSX.Element => {
+    const tagRow = row.value as TagRow;
+    return (
+        <div className='tableCell'>
```
Comment (ErlendLygre) [inline src/modules/Preservation/views/AddScope/SelectTags/SelectTags.tsx:31]: Is there a way to solve this with a styled component? CSS classes should be avoided when using Styled Components

File: src/modules/Preservation/views/AddScope/SelectMigrateTags/SelectMigrateTags.tsx [hunk 2]
```diff
@@ -26,6 +27,51 @@ type SelectMigrateTagsProps = {
     selectedTableRows: Record<string, boolean>;
 }
 
+const getDescriptionColumn = (row: TableOptions<TagMigrationRow>): JSX.Element => {
+    const migrationRow = row.value as TagMigrationRow;
+    return (
+        <div className='tableCell'>
```
Comment (ErlendLygre) [inline src/modules/Preservation/views/AddScope/SelectMigrateTags/SelectMigrateTags.tsx:33]: Is there a way to solve this with a styled component? CSS classes should be avoided when using Styled Components

File: src/modules/Preservation/views/ScopeOverview/ScopeOverview.tsx [hunk 3]
```diff
@@ -271,9 +271,9 @@ const ScopeOverview: React.FC = (): JSX.Element => {
         }));
     }, [filterForProjects]);
 
-    useEffect(() => {
-        refreshScopeList(true);
-    }, [moduleAreaHeight, moduleHeaderHeight]);
+    // useEffect(() => {
```
Comment (ErlendLygre) [inline src/modules/Preservation/views/ScopeOverview/ScopeOverview.tsx:274]: Remove/explain commented out code

File: src/modules/Preservation/views/ScopeOverview/ScopeOverviewTable.tsx [hunk 4]
```diff
@@ -180,9 +187,9 @@ const ScopeOverviewTable = (props: ScopeOverviewTableProps): JSX.Element => {
             width: 180,
             maxWidth: 400,
             minWidth: 150,
-            filter: (rows: UseTableRowProps<PreservedTag>[], id: number, filterType: string): UseTableRowProps<PreservedTag>[] => {
-                return rows.filter((row) => { return row.original.tagNo.toLowerCase().indexOf(filterType.toLowerCase()) > -1; });
-            }
+            // filter: (rows: UseTableRowProps<PreservedTag>[], id: number, filterType: string): UseTableRowProps<PreservedTag>[] => {
```
Comment (ErlendLygre) [inline src/modules/Preservation/views/ScopeOverview/ScopeOverviewTable.tsx:190]: Remove or explain commented out code.

File: src/modules/Preservation/views/ScopeOverview/ScopeOverviewTable.tsx [hunk 5]
```diff
@@ -232,19 +239,19 @@ const ScopeOverviewTable = (props: ScopeOverviewTableProps): JSX.Element => {
...
-                            const searchField = `${row.original.purchaseOrderNo}/${row.original.calloffNo}`;
-                            return searchField.toLowerCase().indexOf(filterType.toLowerCase()) > -1;
-                        } else {
-                            return row.original.purchaseOrderNo.toLowerCase().indexOf(filterType.toLowerCase()) > -1;
-                        }
-                    }
-                }
-                );
-            }
+            // filter: (rows: UseTableRowProps<PreservedTag>[], id: number, filterType: string): UseTableRowProps<PreservedTag>[] => {
```
Comment (ErlendLygre) [inline src/modules/Preservation/views/ScopeOverview/ScopeOverviewTable.tsx:242]: Remove or explain commented out code.

File: src/modules/Preservation/views/ScopeOverview/ScopeOverviewTable.tsx [hunk 6]
```diff
@@ -284,7 +291,7 @@ const ScopeOverviewTable = (props: ScopeOverviewTableProps): JSX.Element => {
             id: 'status',
             Cell: getStatus,
             Filter: SelectColumnFilter,
-            filter: 'text'
+            // filter: 'text'
```
Comment (ErlendLygre) [inline src/modules/Preservation/views/ScopeOverview/ScopeOverviewTable.tsx:294]: Remove or explain commented out code.

File: src/modules/Preservation/views/AddScope/SelectMigrateTags/SelectMigrateTags.tsx (hunk 2 above)
Comment (chormaz) [reply src/modules/Preservation/views/AddScope/SelectMigrateTags/SelectMigrateTags.tsx]: Removed unnecessary div.

File: src/modules/Preservation/views/ScopeOverview/ScopeOverviewTable.tsx (hunk 4 above)
Comment (chormaz) [reply src/modules/Preservation/views/ScopeOverview/ScopeOverviewTable.tsx]: Removed commented-out code.

File: src/modules/Preservation/views/ScopeOverview/ScopeOverviewTable.tsx (hunk 6 above)
Comment (chormaz) [reply src/modules/Preservation/views/ScopeOverview/ScopeOverviewTable.tsx:261]: Removed commented-out code.

File: src/modules/Preservation/views/ScopeOverview/ScopeOverview.tsx (hunk 3 above)
Comment (chormaz) [reply src/modules/Preservation/views/ScopeOverview/ScopeOverview.tsx]: Removed commented-out code.

**Comment by arildbjerkenes** [comment]
When I enter 'Add Scope' -> 'Add tags manually', I get a lot of error in the console log (but things seems to work).

**Review by arildbjerkenes — APPROVED** [review]
There are some UI improvements to make, but I think it's good enough to set in production. You should also look at the error messages in the console.

**Pull request merged**